lib/Model.js

```diff
@@ -85,7 +87,12 @@ class Model {
     /** NPM 6 */
 
     if (parsedOutput.advisories) {
-      Object.values(parsedOutput.advisories).forEach((a) => this.process(a));
+      Object.values(parsedOutput.advisories).forEach((a) => {
+        console.log(a.url.split("/")[4]);
+        // eslint-disable-next-line no-param-reassign, prefer-destructuring
+        a.github_advisory_id = a.url.split("/")[4];
```

Comment:
To not change `this.process`, we add the `github_advisory_id` to the object here.

I though that this would be the easiest to do.

Reply:
Agreed